src/System/Boot.tsx

```diff
@@ -63,6 +65,18 @@ export const Boot: React.FC<
     ...context,
   }
 
+  const unleashConfig = {
+    url: `${sd.UNLEASH_API}frontend/`,
+    clientKey: sd.UNLEASH_CLIENT_KEY,
+    refreshInterval: 15,
+    appName: sd.UNLEASH_APP_NAME,
+    environment: sd.NODE_ENV,
+    context: {
+      userId: props.user?.id,
+      sessionId: sd.SESSION_ID,
+    },
+  }
+
```

Comment:
Considered moving this into its own file, but prefer it here. If you have strong opinions, LMK.

Reply:
A common pattern is to do:

```tsx
import { FlagProvider } from '...'

const FeatureFlagProvider: React.FC<React.PropsWithChildren> = ({ children }) => {
  const { user } = useSystemContext() 
  
  const unleashConfig = {
    url: `${getENV("UNLEASH_API")}frontend/`,
    clientKey: getENV("UNLEASH_CLIENT_KEY"),
    refreshInterval: 15,
    appName: getENV("UNLEASH_APP_NAME"),
    environment: getENV("NODE_ENV"),
    context: {
      userId: user?.id,
      sessionId: getENV("SESSION_ID"),
    },
  }
  
  return <FlagProvider ... >{children}></FlagProvider> 
}

Reply:
Nice! Will make this change.